### ISSUE TITLE
Recaptcha tiles selections

### ISSUE THREAD

**Issue by EsmaeilSaleh** [tridactyl/tridactyl#4560]
I am curious about how we can go around recaptcha tiles. Can we choose the target pictures with tridactyl? 

Great job so far. thanks.

**Comment by bovine3dom** [comment]
I don't think we can. Even if we could it might not be a good idea because mouse movements are used as part of the detection of robots. I assume magically clicking on a tile without moving the mouse at all would be a pretty good indicator that you were a robot, so you might find you end up stuck doing them forever...
